solutions/observability/get-started/logs-essentials.md

```diff
@@ -0,0 +1,132 @@
+---
+navigation_title: Logs Essentials
+applies_to:
+  serverless: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+---
+
+# Get started with {{obs-serverless}} Logs Essentials [logs-essentials-get-started]
+
+```{note}
+Use this guide to get started with the Logs Essentials feature tier of {{obs-serverless}}. Refer to the main [{{observability}} get started](/get-started/index.md) docs to get started with {{obs-serverless}} Complete.
+```
+% Note should link to the feature tier comparison docs once published.
+
+New to Elastic {{obs-serverless}} Logs Essentials? Discover more about its features and how to get started. The following instructions guide you through setting up your first Elastic {{observability}} Logs Essentials deployment, collecting log data, and exploring your data.
+
+## Get started with your use case [get-started-with-use-case]
+
+Learn how to create an {{obs-serverless}} project and use Elastic {{observability}} to gain deeper insight into the behavior of your applications and systems.
+
+:::::::{stepper}
+
+::::::{step} Create an Observability project
+
+An {{obs-serverless}} project allows you to run {{obs-serverless}} in an autoscaled and fully-managed environment, where you don’t have to manage the underlying {{es}} cluster or {{kib}} instances.
+
+::::{dropdown} Steps for creating a project
+:::{note}
+The **Admin** role or higher is required to create projects. Refer to [Assign user roles and privileges](/deploy-manage/users-roles/cloud-organization/manage-users.md#general-assign-user-roles).
+:::
+
+1. Navigate to [cloud.elastic.co](https://cloud.elastic.co/) and log in to your account, or create one.
+1. Select **Create serverless project**.
+1. Under **Elastic for Observability**, select **Next**.
+1. Enter a name for your project.
+1. (Optional) Under **Settings** you can change the following:
+
+    * **Cloud provider**: The cloud platform where you’ll deploy your project. We currently support Amazon Web Services (AWS).
+    * **Region**: The [region](/deploy-manage/deploy/elastic-cloud/regions.md) where your project will live.
+
+1. Select **Edit settings**, and select **Observability Logs Essentials**.
+1. Select **Create serverless project**. It takes a few minutes to create your project.
+1. When the project is ready, click **Continue**.
+
+::::::
```

Comment:
Could you turn steps that are identical across this page and https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2405/solutions/observability/get-started into snippets, and then use the include directive? Easier to maintain that way!